tests/unit/LSYiiValidatorsTest.php

```diff
@@ -27,4 +27,155 @@ public function testHtmlEntityDecodeFilter()
             $this->assertEquals($expected, $actual);
         }
     }
+
+    /**
+     * Testing that the xssfilter attribute will always be
+     * false for super admin users even if filterxsshtml is
+     * changed.
+     */
+    public function testXssFilterAttributeForSuperAdmin()
+    {
+        //Mocking super admin login.
+        \Yii::app()->session['loginID'] = 1;
```

Comment:
We must put a information on manual : For test : we use user #1 as superadmin. 

https://github.com/LimeSurvey/LimeSurvey/blob/a3605854a63d56b360660dea47e144ea766d382f/application/config/config-defaults.php#L673